backend/homework/2/src/main/java/handson/parttwo/StudentUtil.java

```diff
@@ -0,0 +1,98 @@
+package handson.parttwo;
+
+import QuestionTwo.SentimentAnalyzer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.Arrays;
+
+import java.util.ArrayList;
+
+
+
+
+
+import java.util.Scanner;
+
+public class StudentUtil {
+    public static double[] calculateGPA(int[] studentIdList, char[][] studentsGrades) {
+        int numberOfData = studentsGrades.length;
+        double[] gpaCalculated = new double[numberOfData];
+
+        for(int i = 0 ; i < numberOfData ; i++){
```

Comment:
Integrate foreach loops, use constants wherever required like for grades

Reply:
noted